duckduckgo/BrowserServicesKit
# Manage Private Email Addresses

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/72649045549333/1204615490114675/f
iOS PR: https://github.com/duckduckgo/iOS/pull/1828
macOS PR: https://github.com/duckduckgo/macos-browser/pull/1319
What kind of version bump will this require?: Minor

**Optional**:

**Description**:
Allows users to manage their Private Email addresses from the browser.


**Steps to test this PR**:
1. Check macOS/iOS PRs and test functionality
2. Check iOS PR and smoke test password generation and credentials saving




---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
